app/services/proofing/aamva/proofer.rb

```diff
@@ -70,11 +70,12 @@ def build_result_from_response(verification_response)
           transaction_id: verification_response.transaction_locator_id,
           requested_attributes: requested_attributes(verification_response).index_with(1),
           verified_attributes: verified_attributes(verification_response),
+          jurisdiction_in_maintenance_window: jurisdiction_in_maintenance_window?(jurisdiction),
         )
       end
 
       def parse_verification_errors(verification_response)
-        errors = errors = Hash.new { |h, k| h[k] = [] }
+        errors = Hash.new { |h, k| h[k] = [] }
```

Comment:
Happened across this. I _think_ this was an error vs. something very clever I don't understand?